CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Breaking Changes
 
+* `ilab`: **Deprecation of Python 3.9 and Python 3.12** Due to changes to training requiring the usage of [GPTDolomite](https://github.com/instructlab/GPTDolomite), Python versions 3.9 and 3.12 are no longer supported. If you are using either of these versions, you will need to start using either Python 3.10 or Python 3.11 to use this and subsequent versions of the CLI.
```

Comment:
```suggestion
* `ilab`: **Deprecation of Python 3.9 support and withdrawal of Python 3.12 support** Due to changes to training requiring the usage of [GPTDolomite](https://github.com/instructlab/GPTDolomite), Python 3.9 is no longer supported and Python 3.12 support is currently withdrawn. If you are using either of these versions, you will need to start using either Python 3.10 or Python 3.11 to use this and subsequent versions of the CLI.
```

Some suggested wording improvements. I don't think we are deprecating 3.12 support. Rather we are withdrawing support for now. It may be added back in the future (hopefully).